modules/neovim/init/spellcheck.nix

```diff
@@ -25,9 +28,45 @@ in {
         A list of languages that should be used for spellchecking.
 
         To add your own language files, you may place your `spell`
-        directory in either `~/.config/nvim` or the
+        directory in either {file}`~/.config/nvim` or the
         [additionalRuntimePaths](#opt-vim.additionalRuntimePaths)
-        directory provided by **nvf**.
+        directory provided by nvf.
+      '';
+    };
+
+    extraSpellFiles = mkOption {
+      type = attrsOf (listOf str);
+      default = {"en.utf-8" = ["nvf" "word_you_want_to_add"];};
```

Comment:
"word_you_want_to_add" should not be a default

Reply:
This is a testing value, default will be an empty attrset.